src/services/users.service.ts

```diff
@@ -52,6 +52,17 @@ class UserService {
     const deleteUserData = await this.users.delete({ where: { id: userId }, select: { email: true, id: true } });
     return deleteUserData;
   }
+
+  public async manageGroups(userId: number, data: any) {
+    if (isEmpty(data)) throw new HttpException(400, "Groups don't exist");
+    
+    const groups = await this.users.update({
+      where: { id: userId },
+      data
+    })
+
+    return groups;
+  }
```

Comment:
I don't think it's a good idea for the data variable that is manipulated directly in prism to be passed through a function parameter (where in the controller it is received by the request body), **this is extremely dangerous, since a lot of sql code can be passed by the user.**


Change it and also add braces and break lines like mentioned on **findGroupById** method.

On line 56 its important to type the function like we have done on other functions. and type the **data** variable. Its very important so to type the **data** so we can make our application type safe, and avoid unwanted code

On line 57 try to avoid Magic Numbers.